docs/getting-started/playground/start.md

```diff
@@ -0,0 +1,104 @@
+---
+id: start
+---
+
+# Gno Playground
+
+## Overview
+
+The Gno Playground is a web editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
```

Comment:
Not a strong suggestion: we can also call out its similarity to [go playground](https://go.dev/play/)